lib/mayaUsd/ufe/ProxyShapeHierarchy.h

```diff
@@ -77,6 +77,8 @@ class MAYAUSD_CORE_PUBLIC ProxyShapeHierarchy : public Ufe::Hierarchy
     insertChildCmd(const Ufe::SceneItem::Ptr& child, const Ufe::SceneItem::Ptr& pos) override;
 
     Ufe::UndoableCommand::Ptr reorderCmd(const Ufe::SceneItemList& orderedList) const override;
+
+    Ufe::UndoableCommand::Ptr ungroupCmd() const override;
```

Comment:
I think this needs to be #ifdef UFE_V3

Reply:
I already have it protected. See line 82?